tools/interactive/interactivetool_metashark.xml

```diff
@@ -27,7 +27,9 @@
     <help>
 <![CDATA[
 
-`MetaShARK <https://github.com/earnaud/MetaShARK-v2>`_ is an work in progress innovative R Shiny App to automate metadata creation in Ecological Metadata Language. Based on EML Assembly Line R package, MetaShARK allows you to produce EML metadata only from text files as csv or tsv, and allows you to provide metadata on data attributes, geographic, taxonomic and temporal coverages, authors, summary, keywords and methods. Futhermore, the 1.7.3 version alows you to add EML semantic annotation related to attributes names and/or keywords, based on terminological resources from Bioportal. This version of MetaShARK is provided "as is" for testing, as bugs can appear. Infrastructure hosting this app, people and organizations developping this app can't be responsible of any issues or errors from the app.
+`MetaShARK <https://github.com/earnaud/MetaShARK-v2>`_ is an work in progress innovative R Shiny App to automate metadata creation in Ecological Metadata Language. Based on EML Assembly Line R package, MetaShARK allows you to produce EML metadata from text files as .csv (comma as separator) or .tsv (tabular as separator), geographical vector (GeoJSON  and shapefile), geographical raster (GeoTiff) or any kind of other types of data files.  MetaShaRK allows you to provide metadata on data attributes, geographic, taxonomic and temporal coverages, authors, summary, keywords and methods. Futhermore, the 1.7.5 version alows you to add EML semantic annotation related to attributes names and/or keywords, based on terminological resources from Bioportal. This version of MetaShARK is provided "as is" for testing, as bugs can appear. Infrastructure hosting this app, people and organizations developping this app can't be responsible of any issues or errors from the app.
+
+Depending on the operating system and web browser used, attributes level information inference can failed depending on file extensions. We thus recommend to use .csv extension for comma separated files, .tsv for tabular files and .geoJSON for dedicated spatial vector files. 
```

Comment:
For galaxy this does not matter does it?

Reply:
strangely, it seems to matter! I tested on my local Galaxy instance, and I dont have the same behaviour on chrome and firefox... Amazing !! In fact, the datatype / MIME type is not seen the same way from OS and browsers and as we are using these MIMETYPE into the app to guess the datatype...

Reply:
Galaxy has no file file extensions. Or you can set those extensions in the command section and hardcode the extensions. Thus this text is not needed imho if you have the extension under control.

Or do you mean here files loaded into MetaShARK from the user Desktop?

I will merge so that you can work on your tutorial, but think about it and maybe remove it later.